src/test/suite/index.ts

```diff
@@ -6,8 +6,8 @@ export function run(): Promise<void> {
 	// Create the mocha test
 	const mocha = new Mocha({
 		ui: 'tdd',
+    color: true,
 	});
-	mocha.useColors(true);
```

Comment:
useColors was deprecated and now removed completely from Mocha. Replaced with `color` option